cli/cmd/release_create.go

```diff
@@ -146,11 +151,25 @@ func (r *runners) setKOTSDefaultReleaseParams() error {
 	}
 
 	r.args.createReleasePromoteEnsureChannel = true
-
+	
+	// Check if lintReleaseFailOn has been provided, if not set to error
+	// should setup as a function that returns each 
+	if r.args.lintReleaseFailOn == "none" {
+		r.args.lintReleaseFailOn = "none"
+	} else if r.args.lintReleaseFailOn == "info" {
+		r.args.lintReleaseFailOn = "info"
+	} else if r.args.lintReleaseFailOn == "warn" {
+		r.args.lintReleaseFailOn = "warn"
+	} else if r.args.lintReleaseFailOn == "error" {
+		r.args.lintReleaseFailOn = "error"
+	} else {
+		return errors.New("You have entered an incorrect fail-on flag. See 'replicated release create --help' for valid options.")
+	}
```

Comment:
I thought the linter command we're delegating to handles this logic already? Have you tried passing an invalid option, without this logic included?

Reply:
The lint is set to false by default. If ```--lint``` is provided it will only fail on an 'error'.  If the user provides the ```--fail-on``` flag along with ```--lint``` it will check to ensure that they are valid options (i.e. to check for misspelled words).

Reply:
right, but doesn't the downstream command we call [here](https://github.com/replicatedhq/replicated/pull/170/files#diff-dc2fd422e3ea0093695faa8716b4fcb019cb1e2044ffc143cdb3b94380dd1995R219) have that same logic in it [over here](https://github.com/replicatedhq/replicated/blob/master/cli/cmd/release_lint.go#L48-L50)?

Reply:
You are correct. The release_lint.go contains the check for valid ```--fail-on``` options. Removing the ```--fail-on``` checks in the setKOTSDefaultReleaseParams will default fail-on to 'error' when the ```--lint``` flag is provided.

Reply:
okay cool -- so do we still need this logic in here or can it be removed?